rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -152,52 +139,52 @@ void eventLogger_logReleaseBtcRequested_after_rskip_146_185() throws IOException
         verify(repository, never()).transfer(any(), any(), any());
         verify(eventLogger, times(1)).logReleaseBtcRequested(any(byte[].class), any(BtcTransaction.class), any(Coin.class));
         verify(eventLogger, times(1)).logReleaseBtcRequestReceived(any(), any(), any());
-        verify(eventLogger, times(0)).logReleaseBtcRequestRejected(any(), any(), any());
+        verify(eventLogger, never()).logReleaseBtcRequestRejected(any(), any(), any());
     }
 
     @Test
-    void eventLogger_logReleaseBtcRequested_after_rskip_146_185_326() throws IOException {
-        when(activationMock.isActive(ConsensusRule.RSKIP146)).thenReturn(true);
-        when(activationMock.isActive(ConsensusRule.RSKIP185)).thenReturn(true);
-        when(activationMock.isActive(ConsensusRule.RSKIP326)).thenReturn(true);
+    void eventLogger_logReleaseBtcRequested_release_before_papyrus_and_updateCollections_after_papyrus() throws IOException {
```

Comment:
I think this test can be split into two tests, before and after papyrus. Or better, it can be parameterized.

Reply:
I think the idea here is that the release is requested before papyrus and updateCollections is executed after